.github/workflows/ci.yml

```
@@ -0,0 +1,99 @@
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 13: did not find expected key

(syntax-check)